whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/RdapObject.java

```diff
@@ -91,6 +95,20 @@ public String getLang() {
         return lang;
     }
 
+    public List<Ip> getNetworks() {
+        if (networks == null) {
+            networks = Lists.newArrayList();
+        }
+        return networks;
+    }
+
+    public List<Autnum> getAutnums() {
+        if (autnums == null) {
+            autnums = Lists.newArrayList();
+        }
```

Comment:
We use the same class to map other object types , will this have an empty array in json structure for them as well?

Reply:
Attached evidences on the task. Tested with a Domain object